package-deps-hash/.vscode/launch.json

```diff
@@ -1,25 +0,0 @@
-{
-  // Use IntelliSense to learn about possible Node.js debug attributes.
-  // Hover to view descriptions of existing attributes.
-  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
-  "version": "0.2.0",
-  "configurations": [
-    {
-      "type": "node",
-      "request": "launch",
-      "name": "Launch Program",
-      "program": "${workspaceRoot}/node_modules/gulp/bin/gulp.js",
```

Comment:
@nickpape-msft @iclanton Do we need these files? Seems like people are using them